bridge/test/nftbridge/NFTBridge_test.js

```diff
@@ -190,6 +200,126 @@ contract("Bridge NFT", async function(accounts) {
           );
         });
       });
+
+      describe("Fixed Fee", async function() {
+        it("Federator should receive fee", async function() {
+          const fixedFee = new BN("15");
+          let receipt = await this.bridgeNft.setFixedFee(fixedFee, {
+            from: bridgeManager,
+          });
+          utils.checkRcpt(receipt);
+
+          const tokenId = 9;
+          await mintAndApprove(this.token, this.bridgeNft.address, tokenId);
+
+          const tokenOwnerBalance = await utils.getBalance(tokenOwner);
+          const federatorBalance = await utils.getBalance(federation);
+
+          const tx = await this.bridgeNft.receiveTokensTo(
+            this.token.address,
+            anAccount,
+            tokenId,
+            {
+              from: tokenOwner,
+              value: fixedFee,
+            }
+          );
+          utils.checkRcpt(tx);
+
+          const actualTokenOwnerBalance = await utils.getBalance(tokenOwner);

```

Comment:
I'd rename actual to current, here and on every occurrence

Reply:
done at https://github.com/rsksmart/tokenbridge/pull/248/commits/5e908cbcb56c8924846eff80da10efa0adef6d0f